install4j/MonsterUtilities.install4j

```diff
@@ -0,0 +1,465 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<install4j version="8.0" transformSequenceNumber="8">
+  <directoryPresets config="../MonsterUtilities-${compiler:sys.version}.jar" />
+  <application name="MonsterUtilities" applicationId="6836-5354-4913-5566" mediaDir="." shortName="MUtil" publisher="Xerus2000" version="dev" macVolumeId="92204b6f65bce8d0" javaMinVersion="1.8" javaMaxVersion="1.8">
+    <jreBundles jdkProviderId="JBR" />
+  </application>
+  <files>
+    <mountPoints>
+      <mountPoint id="159" />
+    </mountPoints>
+    <entries>
+      <fileEntry mountPoint="159" file="MonsterUtilities-${compiler:sys.version}.jar" />
+    </entries>
+  </files>
+  <launchers>
+    <launcher name="MonsterUtilities" id="58">
+      <executable name="MonsterUtilities" iconSet="true" failOnStderrOutput="false" executableMode="gui">
+        <versionInfo include="true" />
+      </executable>
+      <java mainClass="xerus.monstercat.MainKt">
+        <classPath>
+          <scanDirectory location="." failOnError="false" />
+        </classPath>
+      </java>
+      <iconImageFiles>
+        <file path="/home/defvs/IdeaProjects/monsterutilities/assets/favicon.png" />

```

Comment:
that won't work

Reply:
a12efca6252fb360eb5addaf94977e0376853f5c